packages/tasks/src/model-libraries.ts

```diff
@@ -471,6 +471,12 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		countDownloads: `path:"MeshAnything_350m.pth"`,
 		snippets: snippets.mesh_anything,
 	},
+	mitie: {
+		prettyLabel: "MITIE",
+		repoName: "MITIE",
+		repoUrl: "https://github.com/mit-nlp/MITIE",
+		countDownloads: `path:total_word_feature_extractor.dat`,
```

Comment:
```suggestion
		countDownloads: `path_filename:"total_word_feature_extractor"`,
```

(nit) will work with subfolders